test/src/unit-cppapi-nullable.cc

```diff
@@ -313,8 +313,11 @@ void NullableArrayCppFx::read(
     }
   }
 
+  Subarray sub(ctx_, array);
```

Comment:
Nit: Condense all 3 lines under the comment, which should probably be updated for clarity.